cluster/charts/crossplane/templates/clusterstackinstalls.yaml

```diff
@@ -0,0 +1,9 @@
+{{- range $arg := .Values.clusterStacks }}
+apiVersion: stacks.crossplane.io/v1alpha1
+kind: ClusterStackInstall
+metadata:
+  name: "stack-{{ $arg }}"
+spec:
+  package: "crossplane/stack-{{ $arg }}:master"
```

Comment:
can we make this the stable version of a stack that was available at the time the main crossplane released was made? This will be consumed by GitLab ADO users from stable releases.

Reply:
I think we don't have `stable` tags and all infrastructure stacks have version `v0.1.0`. I'll set tags as `v0.1.0`.